### ISSUE TITLE
Suspecious bug in python.app provided by conda-forge

### ISSUE THREAD

**Issue by easior** [conda-forge/conda-forge.github.io#832]
I installed FEnicsProject provided by conda-forge channel in new Anaconda python Enviroment on MacOS 10.10. Spyder was also installed into it. I found that Spyder can't be started. In order to trace the issue, I execute the following command lines in iTerms:
```sh
$ conda activate fenicsproject
$ spyder
/Users/xxx/Developer/anaconda3/envs/fenicsproject/bin/pythonw: line 3: /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/Contents/MacOS/python: No such file or directory
```
Therefor, I want to see what happend with python.app provided by conda-forge by typing
```sh
$ ls /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/Contents/MacOS/python
ls: /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/Contents/MacOS/python: No such file or directory
$ ls /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/
Contents pythonapp
$ ls /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/Contents/
lib
$ ls /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/pythonapp/Contents/
Info.plist MacOS PkgInfo Resources
```
You see, some directory path in python.app didn't follow Apple's naming convention. So please help me to file this bug and fix it as soon as possible.

Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

**Comment by scopatz** [comment]
I am not a mac user, but I don't believe that conda packages will follow the apple naming convention

**Comment by easior** [comment]
Of course, it's not obligated to follow this mechanism. But for those packages suffixed by .app on MacOS, they should follow. You see, this package is named by python.app located at /Users/xxx/Developer/anaconda3/envs/fenicsproject/. As a comparison, in standard Anaconda Python distribution, there is a similar package named also by python.app located at /Users/xxx/Developer/anaconda3, whose directory path in it DO follow Apple's naming convention.

Last but not the least, the former package caused a suspicious bug, since Spyder can't find the binary file, i.e. python, in the directory /Users/xxx/Developer/anaconda3/envs/fenicsproject/python.app/Contents/MacOS/. So, please fix it, or change some rules in Spyder package.

**Comment by scopatz** [comment]
This really seems like it might be an issue to report on the spyder feedstock, rather than here

**Comment by easior** [comment]
I think this should be the proper place to report it to, because Spyder provided by standard Anaconda Python distribution works well. However, Spyder provided by conda-forge can't work well with python.app also provided by conda-forge. Certainly, if you make sure it happened to Spyder, I could move to there. I hope that is the last place where I should go since I have transferred twice before.

**Comment by scopatz** [comment]
I mean that this is an issue for this repo: https://github.com/conda-forge/spyder-feedstock
